edm4hep/CMakeLists.txt

```diff
@@ -27,6 +27,14 @@ endif()
 PODIO_ADD_ROOT_IO_DICT(edm4hepDict edm4hep "${headers}" src/selection.xml)
 add_library(EDM4HEP::edm4hepDict ALIAS edm4hepDict )
 
+add_library(edm4hep_v2 SHARED schema_evolution/src/OldLinkEvolution.cc)
+target_include_directories(edm4hep_v2 PRIVATE
+  $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/edm4hep/schema_evolution/include>
+)
+target_link_libraries(edm4hep_v2 PUBLIC podio::podio EDM4HEP::edm4hep)
```

Comment:
I am not entirely sure about the naming of this library here. Maybe `v2` gives the wrong impression? Ideally this would simply be merged into `edm4hep` but the problem for that is that we still need a dedicated dictionary at the moment, because the `selection.xml` for EDM4hep is generated and we cannot simply inject the few datatypes that would be necessary into that.

Reply:
Maybe `edm4hepOLE` (OldLinkEvolution)? or `edm4hepOLD` (OldLinkData)?

Reply:
:D that would be some nice backronym in this case. Would be OK for me, but I am not sure we can find one for all future changes.

Reply:
Renamed it to `edm4hepOldSchemas` which should hopefully be a somewhat future proof catch-all.